apps/backend/src/handler/questionHandler.ts

```diff
@@ -113,9 +136,32 @@ const questions = new Hono<Context>()
 		if (annotations && !annotations.includes("")) {
 			annotationsParsed = annotations;
 		}
+		let orderByDir: OrderByDirectionExpression = "desc";
+		if (orderByParsed.length > 0 && orderBy) {
+			if (orderByParsed.startsWith("res")) {
+				orderByParsed = "response_text";
+			} else if (orderByParsed.startsWith("anno")) {
+				orderByParsed = "annotation";
+			} else if (orderByParsed.startsWith("phen")) {
+				orderByParsed = "phenomenon_name";
+			} else if (orderByParsed.startsWith("var")) {
+				orderByParsed = "variety_name";
+			} else if (orderByParsed.startsWith("place")) {
+				orderByParsed = "place_name";
+			} else if (orderByParsed.startsWith("age")) {
+				orderByParsed = "age_group_name";
+			} else if (orderByParsed.startsWith("infid")) {
+				orderByParsed = "comment";
+			} else {
+				orderByParsed = "";
+			}
+			if (dir === "asc") {
+				orderByDir = "asc";
+			}
+		}
 
-		const lowerAgeParsed = Number.isNaN(lowerAge) ? 0 : Number(lowerAge);
-		const upperAgeParsed = Number.isNaN(upperAge) ? 100 : Number(upperAge);
+		const lowerAgeParsed = !Number.isNaN(lowerAge) ? 0 : Number(lowerAge);
```

Comment:
i'm 99% certain this breaks the age filter. I reversed it in my latest PR https://github.com/lapis-project/lapis/pull/105